specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/streams/kafka/subscribe.topic.space/server.rpt

```diff
@@ -0,0 +1,76 @@
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+accept "zilla://streams/kafka0"
+        option zilla:window 8192
+        option zilla:transmission "duplex"
+
+accepted
+
+read zilla:begin.ext ${kafka:matchBeginEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .capabilities("FETCH_ONLY")
+                                .topic("mqtt-sensors")
+                                .filter()
+                                    .headers("zilla:filter")
+                                        .sequence("sensor")
+                                        .sequence("one")
+                                        .build()
+                                    .build()
+                                .evaluation("EAGER")
+                                .build()
+                            .build()}
+
+connected
+
+read advised zilla:flush ${kafka:flushEx()
+                                .typeId(zilla:id("kafka"))
+                                .merged()
+                                  .fetch()
+                                    .capabilities("FETCH_ONLY")
+                                    .filter()
+                                        .headers("zilla:filter")
+                                            .sequence("sensor")
+                                            .sequence("one")
+                                            .build()
+                                        .build()
+                                    .filter()
+                                        .headers("zilla:filter")
+                                            .sequence("sensor")
+                                            .sequence("two")
+                                            .build()
+                                        .build()
+                                    .build()
+                                .build()}
+
+accepted
+
+read zilla:begin.ext ${kafka:matchBeginEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .capabilities("FETCH_ONLY")
+                                .topic("mqtt-devices")
+                                .filter()
+                                    .headers("zilla:filter")
+                                        .sequence("device")
+                                        .skip(1)
+                                        .build()
+                                    .build()
+                                .evaluation("EAGER")
+                                .build()
+                            .build()}
+
+connected
```

Comment:
We'll need to verify subscription ids work properly as messages are delivered from different kafka topics in the topic space, potentially matching topic filters within a specific kafka topic, as well as a higher level filter pattern spanning more than one kafka topic.

This will also come up for `retained` later too as we'll need to retrieve retained messages across the topic space matching the topic filter when we need to replay retained messages on subscribe.